.github/workflows/deploy.yaml

```diff
@@ -59,6 +59,17 @@ jobs:
       - name: push image
         run: docker push --all-tags $DOCKER_IMAGE
 
+      - name: Apply new schema
+        if: github.ref == 'refs/heads/main' || github.ref == 'refs/heads/staging'
+        run: |
+          # cloud run jobs to update
+          run_job_names=("$DEPLOY_TARGET-schema-migrator" "$DEPLOY_TARGET-reference-schema-migrator" "$DEPLOY_TARGET-reference-updater" "$DEPLOY_TARGET-reference-update-hpo-terms" "$DEPLOY_TARGET-reference-import-panels")
+          for cloud_run_target in run_job_names; do
+          
+            gcloud beta run jobs update \
```

Comment:
Should we wait on these jobs to finish successfully within the CI?

Reply:
Yep!